### DISCUSSION TITLE
How to analyze the error？

### DISCUSSION THREAD

**Discussion by changpenghui616** [memtest86plus/memtest86plus#336] [Q&A]
Hi
    It have some error when test bad ddr in PC，so wether have some method to analyze the Error address，in another way，how to contact the address with the ddr die、channel/rank/row/column
PS：I found the program get the physical address in ‘efisetup.c’，and will transfer it to virtual address when test ，but not sure how to find the problem futher
thanks

**Comment by changpenghui616** [comment]
In memtest86 pro， it look support dimm/chip decoding （https://www.memtest86.com/tech_DIMM_Decoding.html）
so ，memtest86plus whether have plan to support this feature

**Comment by debrouxl** [comment]
I'd say that memtest86+ probably wants to support this feature, like a number of others. However, unlike PassMark's commercial offering with a crippled free-to-use version, memtest86+ is developed during our free time as unpaid work (well, it goes for both Martin and I, but I think it goes for Samuel as well). Nobody's working on memtest86+ even remotely close to full time.
As such, there are higher priority tasks to focus on during our low amount of free time, such as:
* SPD reporting: done for most relevant non-server controllers during previous development cycles, but many Xeon processors from the past decade require special hardware configuration not yet implemented by memtest86+, #65 ;
* IMC reporting: done by Sam for most recent desktop IMCs during this development cycle ( #280 ), more to come for laptop/mobile or server platforms;
* EDAC support and reporting of corrected ECC errors: per #92, Sam started work on it.

**Comment by changpenghui616** [comment]
Thank ur reply, I think i understand the status now.
thank ur job （both martin and samuel），i will try to research this problem at spare time too.if have any idea，i will contact to u
thanks again
